src/pages/purchase-orders/create/Create.tsx

```diff
@@ -112,6 +112,10 @@ export default function Create() {
 
       return value;
     });
+
+    return () => {
+      setPurchaseOrder(undefined);
+    };
```

Comment:
@beganovich The same issue occurs with vendor selection when creating a purchase order, and that is the reason for implementing this cleanup.